packages/ddp-server/livedata_server.js

```diff
@@ -1433,6 +1458,11 @@ Server = function (options = {}) {
     // For testing, allow responding to pings to be disabled.
     respondToPings: true,
     defaultPublicationStrategy: publicationStrategies.SERVER_MERGE,
+    // how many messages should we queue during a non-graceful disconnect before we kill the session (to insure against memory leaks)
+    maxMessageQueueLength: 100,
+    // how long we should maintain a session for after a non-graceful disconnect before killing it
+    // sessions that reconnect within this time will be resumed with minimal performance impact.
+    disconnectGracePeriod: 5000,
```

Comment:
Maybe we should up this up to `15000` to copy the heartbeat timeout?

Reply:
This seems reasonable.

Reply:
Synced